build.sbt

```diff
@@ -185,7 +185,8 @@ lazy val tests = crossProject(JVMPlatform, JSPlatform)
   )
   .jsSettings(
     Test / scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.CommonJSModule)),
-    useYarn := true
+    useYarn := true,
+    yarnExtraArgs += "--frozen-lockfile",
```

Comment:
I forgot to add this in the original PR. `--frozen-lockfile` prevents the lockfiles from changing by themselves during CI, which would be bad.